Feature/improve azure function docs (#244)
meshcloud/meshcloud-docs: docs/meshstack.azure.landing-zones.md

subscription and additional specific policies can be defined. A blueprint can be configured to decline users to change or delete the
resources and policies created by the blueprint.

> The [Landing Zone](./meshcloud.landing-zones.md) can be configured in the `Administration` section. If a project is selected to have an Azure location a landingzone must be picked by the user. By choosing a landing zone platform specific configuration can be set (in this case for Azure).

Author: rewrote the paragraph to use active voice too

The next section describe the individual building blocks that platform operators can configure to make up an Azure Landing Zone.

## Management Group Assignment

All newly created [meshProjects](./meshcloud.project.md) get their corresponding Subscription assigned to this [Management Group](https://azure.microsoft.com/en-us/features/management-groups/). The Management Group can be configured by an platform operator through the Azure portal or using infrastructure as code.

> Management Groups used in different Azure [Landing Zones](./meshcloud.landing-zones.md) should not overlap or be nested into one another. A flatter Management Group hierarchy is significantly less complex to manage and thereby greatly reduces the risk of security issues through misconfiguration. However, you can nest Landing Zone Management Groups in other Management Groups controlled outside of meshStack to share common policies between landing zones.

## Blueprint Assignment

Operators can optionally define one or more [Blueprints](https://docs.microsoft.com/en-us/azure/governance/blueprints/overview)
 that meshStack will assign to managed Subscriptions.

### Blueprint Name

> The name of the Blueprint which should get assigned to the project. You can leave it empty, then no Blueprint will get automatically assigned.

Reviewer: ```suggestion
The name of the Blueprint which should get assigned to the project. If left empty,  no Blueprint will get automatically assigned.
```

  <figcaption>Static Parameter usage in Azure Blueprint Panel</figcaption>
</figure>

### Max. Auto Upgrade Blueprint Version

> Blueprints are versioned in Azure and can be managed via the Azure Portal. To avoid the accidental assignment of new (and possibly faulty) Blueprints there is this `Max. Auto Upgrade Blueprint Version` field. If you enter a version identifier here which corresponds to a existing Blueprint version in the Azure portal:

Reviewer: ```suggestion
Blueprints are versioned in Azure and can be managed via the Azure Portal. To avoid the accidental assignment of new (and possibly faulty) Blueprints, meshcloud offers a `Max. Auto Upgrade Blueprint Version` field. If you enter a version identifier here which corresponds to a existing Blueprint version in the Azure portal:
```

Author: official wording is always meshStack (to not confuse the product with the company).

- Existing projects with this Landing Zone will get their Blueprint updated to this version on the next [replication](./meshcloud.tenant.md)
- Newly created projects will get the latest Blueprint version assigned (possibly higher then the version configured here)

### Managed Identity

#### Using System Assigned Managed Identity (SAMI)

> In order to assign [Blueprints](https://docs.microsoft.com/en-us/azure/governance/blueprints/overview) the meshStack replicator needs to be configured with the service principal id of the `Azure Blueprints` app provided by Microsoft.

Reviewer: ```suggestion
In order to assign [Blueprints](https://docs.microsoft.com/en-us/azure/governance/blueprints/overview), the meshStack replicator needs to be configured with the service principal id of the `Azure Blueprints` app provided by Microsoft.
```
